src/Couchbase.Lite.Shared/API/Sync/ReplicatorConfiguration.cs

```diff
@@ -21,6 +21,9 @@
 using System.Security.Cryptography.X509Certificates;
 
 using Couchbase.Lite.Internal.Logging;
+#if COUCHBASE_ENTERPRISE
+using Couchbase.Lite.P2P;
+#endif
```

Comment:
What's this for?

Reply:
Need to use this to check MessageEndpoint Type: https://github.com/couchbase/couchbase-lite-net/pull/1323/files#diff-825d20c5c8cd1686646490f8995630790384cc7ff342c9e1a1a66f481c986a19R292-R296